spec/release.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -e
+
+# check if there are any changes in spec in the latest commit
+if git diff --name-only --exit-code HEAD~1 HEAD 'spec/*.json' >> /dev/null; then
+  echo "no changes in spec detected, skipping publishing spec"
+  exit 0
+fi
+
+# "mangle" name to not overwrite for someone running locally
+WEBSITE_DIR=${WEBSITE_DIR:-$HOME/website-123412}
+
+if [[ -d $WEBSITE_DIR ]]; then
+  echo "deleting $WEBSITE_DIR"
+  rm -rf $WEBSITE_DIR
```

Comment:
This looks dangerous.
Do we need to delete it? Can the git command do the right thing if it exists already? If it exists and is not a git repo, we should probably fail.